sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p><code>Lets</code> -&gt; <code>Let's</code> ﻿</p><p> </p><p><code>Reatail</code>  -&gt; <code>Retail</code>  </p><p> </p><p>Please proofread your work before submitting a PR.</p><p> </p><p>What is the source of this data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>